### ISSUE TITLE
Instructions on how to boot from external sdcard

### ISSUE THREAD

**Issue by mda0** [lgeek/okreader#10]
Instructions on how to install on and boot from the external sdcard would be very welcome! Thank you!

**Comment by nguha** [comment]
@lgeek Could you please complete this task?

After checking your [supported hardware table](https://github.com/lgeek/okreader#supported-hardware) I got myself a Kobo Aura (N514). I have okreader compiled and ready to flash. Unfortunately, after opening my Kobo Aura, I realized it has an eMMC chip soldered to the PCB.
According to [your comment on #8](https://github.com/lgeek/okreader/issues/8#issuecomment-303104699), it seems that you are running okreader on a Kobo Aura from the external sdcard. Could you please add instructions on how to prepare such card?

**Comment by klundry** [comment]
Yes, please some detail on the steps needed to write the necessary files to sd card and boot from the external slot on the Aura would be great. I have one here and am willing to test and help with the write up if given something to go on. Thanks.

**Comment by Velsoth** [comment]
I believe this might be of your interest. Any feedback is appreciated.

https://github.com/lgeek/okreader/pull/22

**Comment by klundry** [comment]
> 
> 
> I believe this might be of your interest. Any feedback is appreciated.
> 
> #22

Hey, thanks for this! I haven't been able to give it a go yet but I'm planning to soon and will let you know how it goes and any feedback I have.
